front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -33,7 +48,7 @@ const ScenarioHeader = ({ scenario, infra }: ScenarioHeaderProps) => {
   const scenarioNameRef = useRef<HTMLSpanElement>(null);
   const usernameRef = useRef<HTMLButtonElement>(null);
 
-  const boards = ['Traits', 'Map', 'Macro', 'STD', 'SDD', 'Table', 'Conflicts'];
+  const boards = ['Trains', 'Map', 'Macro', 'STD', 'SDD', 'Table', 'Conflicts'];
```

Comment:
Could we extract this as a const and use it in both `const [activeBoards, setActiveBoards] = useState(…)` above and here?